docs/development/affiliated-packages.rst

```diff
@@ -330,13 +330,12 @@ files manually`_ section since this explains what many of the files do.
 
 #. Decide what license you want to use to release your source code. If
    you don't care and/or are fine with the Astropy license, just edit
-   the file ``licenses/LICENSE.rst`` with your name (or your
-   collaboration's name) at the top as the licensees. Otherwise, make
-   sure to replace that file with whatever license you prefer, and
-   update the ``license`` variable in ``setup.cfg`` to reflect your
-   choice of license. You also may need to update the comment at the
-   top of ``<packagename>/__init__.py`` to reflect your choice of
-   license.
+   the file ``LICENSE.rst`` with your name (or your collaboration's
```

Comment:
Ah, I missed that this refers to the package-template repository. Even if it decided to move the licence file there, it should not be a part of this PR.